recipes/g3log/config.yml

```diff
@@ -0,0 +1,8 @@
+versions:
+  "1.3.2-86":
```

Comment:
What is the difference to `1.3.2`  and why do you want this?

Reply:
The g3log project has the last years been calculating their version number in CMakeLists.txt, based on some git properties, see [https://github.com/KjellKod/g3log/blob/master/CMakeLists.txt#L94](url), line 94-103. The last GitHub release, 1.3.2, is over two years old, and I don't know if they are doing those kind of releases anymore, or if they just stick to their current version scheme. At work we need a newer version than the two year old 1.3.2. There has been a lot of fixes and improvements and we use version 1.3.2-86 from Mars 6, 2020. That's why I added this version.

Reply:
https://github.com/KjellKod/g3log/issues/338

Reply:
@Croydon The g3log project is officially in support mode. Meaning, pull requests will be considered and merged if required, but the author is not going to fix bugs himself anymore. The age of the release is irrelevant, given the recent commits are still. As @arnesor mentioned, the project is only doing support releases, and the age is indicative of that.  The commits in master are untested and potentially dangerous to have in a production environment. If you are bold enough, you can test them, but I highly suggest you don't.